### ISSUE TITLE
Indentation of analysis.sh

### ISSUE THREAD

**Issue by michaelgruenstaeudl** [daisieh/plann#1]
Regarding file _analysis.sh_:
I cannot run your code unless I add proper linebreaks to the if-statements in your code (i.e., for checking (a) if translations start with M, and (b) if new and old translations are identical). Otherwise, bash cannot execute them properly. For example, it should read:

```
for GBF in *.gbf
do
    echo "Looking at proteins in $GBF:"
    if grep -E '/translation="[^M]' $GBF
        then
        echo "ERRORS! $GBF has errors in translation."
        else
        echo "$GBF is fine."
    fi;
done;
```

See the linebreak after `then` and `else`

_EDIT_: Title should be "Linebreaks in analysis.sh"

**Comment by daisieh** [comment]
(sorry it took me so long to get to this, somehow I missed the Github notification!)

This is interesting: my version of bash has never had a problem executing these, but I see no problem with putting them in. They'll be in the next release.

**Comment by daisieh** [comment]
Fixed in #3
